apps/explorer/app/page.tsx

```diff
@@ -24,22 +24,10 @@ export function generateMetadata(): Metadata {
 export const revalidate = 0
 
 // Cache our top hosts fetch and ranking. Revalidate every 24 hours.
-const getCachedTopHosts = unstable_cache(
-  async () => {
-    const [hosts, hostsError] = await to(
-      explored.hostsList({
-        params: { sortBy: 'date_created', dir: 'asc', limit: 500 },
-        data: { online: true },
-      })
-    )
-    if (hostsError) return []
-    return rankHosts(hosts)
-      .slice(0, 5) // Select the top 5.
-      .map((result) => result.host) // Strip score key.
-  },
-  ['top-hosts'],
-  { revalidate: 86400 }
-)
+const getCachedTopHosts = unstable_cache(getCachedTopHostsFunc, ['top-hosts'], {
+  tags: ['top-hosts'],
+  revalidate: 86400,
```

Comment:
This could still run into a cold start issue where it cant get the very first hosts list because of syncing and then has to wait 24 hrs. 5min or something low is fine, its not so expensive of an operation, just don't want it running on every request.